app/src/main/java/blockeq/com/stellarwallet/activities/SendToAddressActivity.kt

```diff
@@ -0,0 +1,63 @@
+package blockeq.com.stellarwallet.activities
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.animation.AnimationUtils
+import android.widget.Toast
+import blockeq.com.stellarwallet.R
+import blockeq.com.stellarwallet.helpers.Constants.Companion.STELLAR_ADDRESS_LENGTH
+import blockeq.com.stellarwallet.services.networking.Horizon.Companion.getBalance
+import com.google.zxing.integration.android.IntentIntegrator
+import kotlinx.android.synthetic.main.content_send_to_address.*
+
+
+class SendToAddressActivity : BasePopupActivity() {
+
+    val ADDRESS_DATA = "ADDRESS"
+
+    override fun setContent(): Int {
+        return R.layout.content_send_to_address
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        titleText.text = "Available: " + getBalance() + " XLM"
+
+        camera_image_button.setOnClickListener { initiateScan() }
+
+        next_button.setOnClickListener {
+            val address = addressEditText.text.toString()
+            if (address.length == STELLAR_ADDRESS_LENGTH) {
+                val intent = Intent(this, SendActivity::class.java).apply {
+                    putExtra(ADDRESS_DATA, address)
+                }
+                startActivity(intent)
+
+                this.overridePendingTransition(R.anim.slide_in_up, R.anim.stay)
+            } else {
+                // Shake animation on the text
+                val shakeAnimation = AnimationUtils.loadAnimation(this, R.anim.shake)
+                addressLayout.startAnimation(shakeAnimation)
```

Comment:
I like the idea of the animation to bring to user's attention. However, is it possible to apply animation to just `addressEditText`? With the animation on `addressLayout` the container seems to be moving off the screen.

Reply:
Yup we can totally do that!